genie-common/src/main/java/com/netflix/genie/common/dto/JobStatusMessage.java

```diff
@@ -67,6 +67,39 @@
     public static final String JOB_PROCESS_NOT_FOUND = "Couldn't check job process status.";
 
     /**
+     * Job PID terminated, exist status is yet to be verified.
+     */
+    public static final String PROCESS_DETECTED_TO_BE_COMPLETE = "Process detected to be complete";
```

Comment:
Seems to me this class should be named `JobStatusMessages.java`. Just nitpicky but more accurate.